src/services/routes/member.routes.ts

```diff
@@ -185,5 +184,5 @@ export default function registerRoutes(
     },
   });
 
-  done();
+  await fastify.register(memberPublicKeyRoutes, { ...opts, prefix: '/:memberId/public-keys' });
```

Comment:
Not completely happy with the way this looks. Maybe we could change this a bit and create some kind of chain function that returns the fastify instance. and then we could construct the paths in one place

would look something like this
```
const orgFastify = registerOrgRoutes(fastifyInstance, opts);
const memberFastify = registerMemberRoutes(orgFastify, opts, prefix: '/:memberId/public-keys');
registerPublicKeyRoutes(memberFastify, opts, prefix: 'public-keys');
registerMemberTokenRoutes(memberFastify, opts, prefix: 'public-keys');
```

Reply:
> Not completely happy with the way this looks

I think I understand the alternative you're suggesting, but can you please elaborate on what you don't like? I don't know what problem we're trying to solve.

As for the alternative itself, correct me if I'm wrong, but we'd be creating a bunch of functions that would only ever be called once, only to wrap one-liners like this? It feels overly complicated.